games/a.yaml

```diff
@@ -167,7 +167,7 @@
   - Jack in the Dark
   - Alone in the Dark 3
   meta:
-    genre: [horror]
```

Comment:
I think its important to keep the horror/survival horror genre as a main label. There really is a huge gap between what would be an action-adventure game like Legend of Zelda and games like Alone in the Dark and Resident Evil when it comes to gameplay features. I know there are very few horror games currently in the list (2 I think) but my vote is that we keep them in their own subset as much as possible.